astropy/modeling/core.py

```diff
@@ -70,7 +70,7 @@ class _ModelMeta(abc.ABCMeta):
     Parameter descriptors declared at the class-level of Model subclasses.
     """
     @classmethod
-    def __prepare__(mcls, name, bases):
+    def __prepare__(mcls, name, bases, **kwds):
```

Comment:
Per https://github.com/astropy/astropy/pull/11098#discussion_r532220641 this will have to be rebased on #11098 and remove `__prepare__`.  Alternatively, if this PR is merged first I can rebase mine.  Either way.

Reply:
Thanks! I've rebased. `__prepare__` is gone.